tests/integration/test_playstation.py

```diff
@@ -95,13 +97,54 @@ def test_playstation_with_feature_flag(
     assert response.ok
 
     outcomes = outcomes_consumer.get_outcomes()
-    assert len(outcomes) == 2
-    assert outcomes[0]["reason"] == "no_event_payload"
-    assert outcomes[1]["reason"] == "no_event_payload"
+    assert len(outcomes) == 0
+
+    while True:
+        _, message = attachments_consumer.get_message()
+        if message is None or message["type"] != "attachment_chunk":
+            event = message
+            break
```

Comment:
Would be a nice cleanup in the future, adding a utility to the `attachments_consumer` that does that for you, looks like a loop like this is used quite a bit throughout the tests.

Reply:
Yeah this is very strange because the moment you add an attachment both the attachment and event land in the `attachments_consumer` rather than the event landing in the `event_consumer`.